README.md

```diff
@@ -1,43 +1,153 @@
+        <li><a href="#wireframe">Wireframe</a></li>
+      </ul>
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#roadmap">Roadmap</a></li>
+    <li><a href="#contributing">Contributing</a></li>
+    <li><a href="#license">License</a></li>
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
 
-## Description
+<!-- ABOUT THE PROJECT -->
+## About The Project
+This app build with the existing knowledge of the React basics, and give you further practice working with codebase linting, component testing, Redux Toolkit, and TypeScript. A user should, at the bare minimum: exercise (add exercise, [pushup, pull, number, completed/not] save to storage , loop)
+* Create complex workouts easily
+* Define your own exercises with custom durations
+* Add , update and remove exercises 
 
+This app used Redux Toolkit and state slices to manage any state data needed across multiple components. In the case where data is created and used only within one component (or possibly one and a few of its direct children), consider using useState() or useReducer() and traditional prop drilling. Most apps will use a combination of "local" state management (useState() and useReducer()) and "global" state management (Redux or Context API).
 
-## Global requirements
-1. authentication/authorization
-2. manage exercises (crud)
+Apply the TypeScript static typing system. try to use TypeScript for the majority of React components! installed typescript, add support for TypeScript linting, and use either .ts or .tsx extension for the files to apply TypeScript type-checking on.
 
-### Import requirements
-1. exercise (add exercise, [pushup, pull, number, completed/not] save to storage , loop)
-2. exercise log
+Use the `BLANK_README.md` to get started.
 
-## highlight important things for next time
-try to make it detailed
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+### Screenshots
 
-guide
-descript, feature, basic layout, functionality
+### Built With
 
-1. feature: add new, exercise, layout, functionality: add button, exercise type, reps, done/not
+A few of the frameworks/libraries that were used to build the project are:
 
-2.
-3.
-4.
+* [![React][React.js]][React-url]
+* Firebase
+* React Redux 
+* Axios
+* React Router
+* EsLint and AirBnb
+* Typescript
 
-Creating the workout scheme in React
-In our app, we want the user to be able to:
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-Add and remove exercises
-Add and remove sets
-For each set, add weight and reps
-For each set, mark as finished or unfinished
+### Wireframe using Figma 
+  ### Getting Started The App build out with simple wireframes using Figma.
+<img src="/assets/images/homewireframe.png"
+        alt="home page"
+             height="80">
+<img src="/assets/images/signinwireframe.png"
+        alt="sign in page"
+             height="80">
+<img src="/assets/images/signupwireframe.png"
+        alt="sign up page"
+             height="80">
```

Comment:
The wireframe images aren't loading for me. Most likely because this file lives at a level where no `/assets/images` folder exists.